### ISSUE TITLE
Respect Smart Moving shift

### ISSUE THREAD

**Issue by Eufranio** [xsun2001/Applied-Energistics-2-Unofficial#8]
To break a cable, we have to shift click the cables using a wrench. When Smart Moving [mod] is present, we can't break the cables by using shift while flying or etc,

**Comment by EverNife** [comment]
Lol... 
That seems to be more like a Smart Moving bug that changes the way "shift" work...

Does other things that required SHIFT works well? (Plugins and other mods and stuffs)

**Comment by Eufranio** [comment]
No, other mods **didn't** respect SmartMoving's shift, but we made a little mod that passes the shift just like vanilla mc does, and AE2 still not accepting it. Is there something that **needs** to be in my code to shift work?

**Comment by xsun2001** [comment]
I am just going to work for this issue. But in my direct mind, it is a sm's problem. I am trying to find the code that process the shirt key. ....
